{"changed":true,"filter":false,"title":"Search Results","tooltip":"/.c9/searchresults","value":"Searching for \u0001Illuminate\\Foundation\\Auth\\User\u0001 in\u0001/kadai-tasklist\u0001 (Found 26 matches in 3 files)\n\n/kadai-tasklist/app/User.php:\n\t6: use Illuminate\\Foundation\\Auth\\User as Authenticatable;\n\n/kadai-tasklist/storage/logs/laravel.log:\n\t4137: [2018-06-07 12:14:22] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t4665: [2018-06-07 13:26:43] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t4740: [2018-06-07 13:28:42] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t4815: [2018-06-07 13:33:23] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t4890: [2018-06-07 13:43:21] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t4965: [2018-06-07 14:31:31] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t5040: [2018-06-07 14:35:08] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t5115: [2018-06-07 14:35:19] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t5190: [2018-06-07 14:45:16] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t5265: [2018-06-07 14:45:42] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t5340: [2018-06-07 14:48:18] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t5415: [2018-06-07 14:50:20] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t5490: [2018-06-07 14:58:10] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t5565: [2018-06-07 14:59:31] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t5637: [2018-06-07 15:00:29] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t5712: [2018-06-07 15:04:49] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t5787: [2018-06-07 15:07:17] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t5862: [2018-06-07 15:07:41] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t5937: [2018-06-07 15:10:35] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t6012: [2018-06-07 15:11:06] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t6087: [2018-06-07 15:11:38] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t6162: [2018-06-07 15:12:02] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t6268: [2018-06-07 15:24:28] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\t6416: [2018-06-07 15:29:47] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)\n\n/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:\n\t4: use Illuminate\\Foundation\\Auth\\User as Authenticatable;\n\nFound 26 matches in 3 files \n\n","undoManager":{"mark":-2,"position":2,"stack":[[{"start":{"row":0,"column":0},"end":{"row":0,"column":1},"action":"insert","lines":[" "],"id":1}],[{"start":{"row":0,"column":0},"end":{"row":0,"column":1},"action":"remove","lines":[" "],"id":2},{"start":{"row":0,"column":0},"end":{"row":2,"column":0},"action":"insert","lines":["Searching for \u0001Illuminate\\Foundation\\Auth\\User\u0001 in\u0001/kadai-tasklist\u0001","",""]},{"start":{"row":2,"column":0},"end":{"row":3,"column":0},"action":"insert","lines":["/kadai-tasklist/app/User.php:",""]},{"start":{"row":3,"column":0},"end":{"row":5,"column":0},"action":"insert","lines":["\t6: use Illuminate\\Foundation\\Auth\\User as Authenticatable;","",""]},{"start":{"row":5,"column":0},"end":{"row":6,"column":0},"action":"insert","lines":["/kadai-tasklist/storage/logs/laravel.log:",""]},{"start":{"row":6,"column":0},"end":{"row":31,"column":0},"action":"insert","lines":["\t4137: [2018-06-07 12:14:22] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t4665: [2018-06-07 13:26:43] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t4740: [2018-06-07 13:28:42] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t4815: [2018-06-07 13:33:23] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t4890: [2018-06-07 13:43:21] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t4965: [2018-06-07 14:31:31] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t5040: [2018-06-07 14:35:08] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t5115: [2018-06-07 14:35:19] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t5190: [2018-06-07 14:45:16] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t5265: [2018-06-07 14:45:42] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t5340: [2018-06-07 14:48:18] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t5415: [2018-06-07 14:50:20] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t5490: [2018-06-07 14:58:10] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t5565: [2018-06-07 14:59:31] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t5637: [2018-06-07 15:00:29] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t5712: [2018-06-07 15:04:49] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t5787: [2018-06-07 15:07:17] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t5862: [2018-06-07 15:07:41] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t5937: [2018-06-07 15:10:35] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t6012: [2018-06-07 15:11:06] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t6087: [2018-06-07 15:11:38] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t6162: [2018-06-07 15:12:02] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t6268: [2018-06-07 15:24:28] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","\t6416: [2018-06-07 15:29:47] local.ERROR: Class 'Illuminate\\Foundation\\Auth\\User' not found {\"exception\":\"[object] (Symfony\\\\Component\\\\Debug\\\\Exception\\\\FatalErrorException(code: 1): Class 'Illuminate\\\\Foundation\\\\Auth\\\\User' not found at /home/ec2-user/environment/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:5)","",""]},{"start":{"row":31,"column":0},"end":{"row":32,"column":0},"action":"insert","lines":["/kadai-tasklist/vendor/laravel/framework/src/Illuminate/Foundation/Auth/User.php:",""]},{"start":{"row":32,"column":0},"end":{"row":34,"column":0},"action":"insert","lines":["\t4: use Illuminate\\Foundation\\Auth\\User as Authenticatable;","",""]},{"start":{"row":34,"column":0},"end":{"row":35,"column":0},"action":"insert","lines":["Found 26 matches in 3 files ",""]}],[{"start":{"row":35,"column":0},"end":{"row":36,"column":0},"action":"insert","lines":["",""],"id":3},{"start":{"row":0,"column":67},"end":{"row":0,"column":97},"action":"insert","lines":[" (Found 26 matches in 3 files)"]}]]},"ace":{"folds":[],"customSyntax":"c9search","scrolltop":446.5,"scrollleft":4.5,"selection":{"start":{"row":36,"column":0},"end":{"row":36,"column":0},"isBackwards":false},"options":{"tabSize":4,"useSoftTabs":true,"useWrapMode":false,"wrapToView":true},"firstLineState":{"row":26,"state":["start",{}],"mode":"ace/mode/c9search"}},"timestamp":1528353056593}